src/components/demos/write/LoginForm.js

```diff
@@ -0,0 +1,123 @@
+  const [someUsernameTyped, setSomeUsernameTyped] = useState(false)
+  const [somePasswordTyped, setSomePasswordTyped] = useState(false)
+
+  const usernameUpperBound = 24
+  const passwordUpperBound = 16
+
+  const noUsernameEntered = usernameInput.length === 0 && someUsernameTyped
+  const usernameAtMaxLength = usernameInput.length === usernameUpperBound
+  const noPasswordEntered = passwordInput.length === 0 && somePasswordTyped
+  const passwordAtMaxLength = passwordInput.length === passwordUpperBound
+
+  const enterUsernameAndPassword = async (e) => {
+    setSomeUsernameTyped(true)
+    setSomePasswordTyped(true)
+    e.preventDefault()
+
+    if (usernameInput.length > 0 && passwordInput.length > 0) {
+      setUsernamePasswordPairs((prev) => [
+        ...prev,
+        usernameInput,
+        passwordInput,
+      ])
+    }
+  }
+
+  const UsernameErrorWarning = () => (
+    <>
+      {noUsernameEntered && <>Cannot be empty</>}
+      {usernameAtMaxLength && <>Maximum length reached</>}
+    </>
+  )
+
+  const PasswordErrorWarning = () => (
+    <>
+      {noPasswordEntered && <>Cannot be empty</>}
+      {passwordAtMaxLength && <>Maximum length reached</>}
+    </>
+  )
+
+  useEffect(() => {
+    if (usernameInput.length > 0) {
+      setSomeUsernameTyped(true)
+    }
+    if (passwordInput.length > 0) {
+      setSomePasswordTyped(true)
+    }
+  }, [usernameInput, passwordInput])
+
+  return (
+    <form onSubmit={enterUsernameAndPassword}>
+      <Input
+        label={'Username'}
+        theme={demoState.theme.form}
+        setInputState={setUsernameInput}
+        upperBound={usernameUpperBound}
+        showInputError={usernameAtMaxLength || noUsernameEntered}
+        InputErrorWarning={UsernameErrorWarning}
+        cySelector={'username'}
+      />
+      <Input
+        label={'Password'}
+        theme={demoState.theme.form}
+        setInputState={setPasswordInput}
+        upperBound={passwordUpperBound}
+        inputType={'password'}
+        showInputError={passwordAtMaxLength || noPasswordEntered}
+        InputErrorWarning={PasswordErrorWarning}
+        cySelector={'password'}
+      />
+      <Container
+        size={10}
+        styles={{
+          flexDirection: 'column',
+          alignItems: 'center',
+          gap: '30px',
+        }}
+      >
+        <Button
+          {...demoState.theme.form.loginButton}
+          data-cy={'login'}
+          type={'submit'}
+          disabled={showSpinner}
+        >
+          {showSpinner ? <Spinner /> : `Login`}
+        </Button>
+        <LoginAttemptText
+          {...demoState.theme.form.loginAttempt}
+          visibility={
+            failedLogin(apiOutput) || loginError(apiOutput)
+              ? 'visible'
+              : 'hidden'
+          }
+          data-cy={'login-attempt'}
+        >
+          {failedLogin(apiOutput) && `Incorrect username or password`}
+          {loginError(apiOutput) && `Hack attempt detected - locked out!`}
```

Comment:
I think the copy and styling could be better on this but it's the right approach

Reply:
changed the hack attempt one to `Suspicious activity detected - account locked`

Reply:
and fixed some broken font css so fonts display correctly for Morello - see updated screenshots in PR description